src/v/utils/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ v_cc_library(
     vint.cc
```

Comment:
>- millisecond precision for refill - to have token refill resolution
at a scale comparable with how frequently the bucket can be used by
different connections (token_bucket_rate_tracker has seconds resolution)

Excellent commit message. Thanks for all the detail!